crates/polars-plan/src/dsl/function_expr/pow.rs

```diff
@@ -24,15 +24,52 @@ impl Display for PowFunction {
     }
 }
 
-fn pow_on_floats<T>(base: &ChunkedArray<T>, exponent: &Series) -> PolarsResult<Option<Series>>
+fn pow_on_chunked_arrays<T, F>(
+    base: &ChunkedArray<T>,
+    exponent: &ChunkedArray<F>,
+) -> PolarsResult<Option<Series>>
+where
+    T: PolarsNumericType,
+    F: PolarsNumericType,
+    T::Native: num::pow::Pow<F::Native, Output = T::Native> + ToPrimitive,
+    ChunkedArray<T>: IntoSeries,
+{
+    if (base.len() == 1) && (exponent.len() != 1) {
+        let base = base
+            .get(0)
+            .ok_or_else(|| polars_err!(ComputeError: "base is null"))?;
+
+        Ok(Some(
+            exponent
+                .into_iter()
+                .map(|exp| exp.map(|exp| Pow::pow(base, exp)))
+                .collect_trusted::<ChunkedArray<T>>()
+                .into_series(),
+        ))
+    } else {
+        Ok(Some(
```

Comment:
Can we `allign` chunks and then use a kernel? For the same reason, iterating over chunkedarrays is slow.

Reply:
thanks for your review! have given this a go